app/src/main/res/values/strings.xml

```diff
@@ -153,18 +153,18 @@
     <string name="progress_title">Working…</string>
     <string name="protection">Protection</string>
     <string name="read_only">Read-only</string>
-    <string name="read_only_warning">KeePass DX does not have permission to write to your database location, so it will be opened read-only.</string>
+    <string name="read_only_warning">Grant KeePass DX write permission to change anything in your database.</string>
```

Comment:
It is possible to manually grant permissions. I am unsure what the problem is.

Reply:
Maybe in your case, but if you access a remote disk read-only and we are not admin, we do not have permissions because it is locked by the administrator and we do not have the power to increase them.